examples/csv-example/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+    kotlin("jvm") version "1.6.10"
+    kotlin("plugin.serialization") version "1.6.10"
+}
+
+//kotlin {
+//    freeCompilerArgs = freeCompilerArgs + "-Xopt-in=kotlin.RequiresOptIn"
+//}
+
+dependencies {
+    implementation("org.cqfn.save:save-adapters:0.1.0-SNAPSHOT")
```

Comment:
you made the release, btw?

Reply:
Not yet, will try to add snapshot publication in #7

Reply:
ok, let's be quick with it